### DISCUSSION TITLE
how to make other side slash

### DISCUSSION THREAD

**Discussion by doyulan** [community/community#80529] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

I can't make a other side slash in my laptop

**Comment by Develops01** [comment]
if you asking for this \ / you can find it on the same key as the pipe (|) symbol and just above the Enter key
or if there is problem with your keyboard you can use virtual keyboard or simply copy that

**Comment by dmpapageo** [comment]
Are you using a regular QWERTY keyboard? 

I used to work with a QWERTZ German keyboard so the special characters were scattered 😂

Let me know, we can sort this out!
